Library.Tests/UnitTests/IssueTests.cs

```diff
@@ -0,0 +1,203 @@
+using Library.Tests.UnitTests;
+using Library.Domain.Models;
+using Xunit;
+
+namespace Library.Tests.UnitTests;
+
+/// <summary>
+/// Unit tests for querying library data using LINQ over in-memory seed collections.
+/// </summary>
+/// <remarks>
+/// Covers the following scenarios:
+/// 1) Issued books ordered by title;
+/// 2) Top-5 readers by number of books in a given period (last 6 months);
+/// 3) Readers who took books for the longest period, ordered by full name;
+/// 4) Top-5 publishers by number of issued books in the last year;
+/// 5) Top-5 least popular books in the last year.
+/// </remarks>
+public class LibraryQueriesTests(DataSeed seed) : IClassFixture<DataSeed>
+{
+    /// <summary>
+    /// Returns distinct issued book titles ordered alphabetically.
+    /// </summary>
+    [Fact]
+    public void IssuedBooks_ShouldBeOrderedByTitle()
```

Comment:
Поправить именование методов. Должен быть PascalCase. Тут и в других методах.
```suggestion
    public void IssuedBooksShouldBeOrderedByTitle()
```